services/gitea/9.3.0/defaults/cm.yaml

```diff
@@ -37,6 +37,17 @@ data:
         service:
           REQUIRE_SIGNIN_VIEW: false
           DISABLE_REGISTRATION: true
+        session:
```

Comment:
i was looking at a bundle after the upgrade and it looks like the default redis deployment is a statefulset with 6 replicas. I'd be a bit concerned about the resource usage increase (mainly for pre-prov), we should sync about this 🤔

Reply:
for now, I've changed it to 3, I'll also check it with the team

Reply:
apparently it is not possible to install it with 3 nodes. this is logs from the gitea-redis-cluster pods when I tried to install it with 3 nodes.   
```shell
*** ERROR: Invalid configuration for cluster creation.
*** Redis Cluster requires at least 3 master nodes.
*** This is not possible with 3 nodes and 1 replicas per node.
*** At least 6 nodes are required.
```

Reply:
using nodes=4 or 5 also resulted in redis deployment failure!

Reply:
oh i see because they are master/replica sets, sounds good then.